post-types/templates/single-tabular.php

```diff
@@ -104,14 +104,15 @@
 				array_push($filter_fields,'"extras_' . $key . '":"'.$selected_param.'"');
 			}
 		endforeach;
-		
+
 		foreach ($filters_datatables_list_array as $key => $resource_id):
 			$selected_param = !empty($_GET[$key]) ? $_GET[$key] : null;
 			if (isset($selected_param)	&& $selected_param !== "all") {
 				array_push($filter_fields,'"extras_' . $key . '":"'.$selected_param.'"');
 			}
 		endforeach;
 		$attrs['filter_fields'] = '{' . implode($filter_fields,",") . '}';
+		$attrs['limit'] = 500;
```

Comment:
Please delete this line. It did for local testing because it take long time in the localhost to load them all.